test/lsVersion.ts

```diff
@@ -0,0 +1,73 @@
+import { StatusBar, VSBrowser, Workbench } from "vscode-extension-tester";
+import { expect } from "chai";
+import * as path from "path";
+import { getStatusBarItem } from "./getStatusBarItem";
+import { homedir } from "os";
+
+describe("LS version", function () {
+  this.timeout(50000);
+
+  it("Checks correct scarb precedence", async function () {
+    // asdf is in fact in PATH and in our tests it is first scarb in  path, special case this.
+    // It is caused by `@actions/core addPath` implementation.
+    // See: https://github.com/actions/toolkit/blob/01f21badd5a7522507f84558503b56c4deec5326/packages/core/src/core.ts#L107
+    if (process.env.PATH_SCARB_VERSION && process.env.ASDF_SCARB_VERSION) {
+      process.env.PATH_SCARB_VERSION = process.env.ASDF_SCARB_VERSION;
+    }
+
+    const scarbs = [
+      // Order is important here.
+      process.env.CONFIG_SCARB_VERSION,
+      process.env.PATH_SCARB_VERSION,
+      process.env.ASDF_SCARB_VERSION,
+    ];
+
+    const expectedScarbVersion = scarbs.find(Boolean); // Find first with value.
+
+    // Ignore this test locally, it is strictly designed for our CI.
+    if (!expectedScarbVersion) {
+      this.skip();
+    }
+
+    if (process.env.CONFIG_SCARB_VERSION) {
+      const workbench = new Workbench();
+
+      const settings = await workbench.openSettings();
+
+      const setting = await settings.findSetting("Scarb Path", "Cairo1");
+
+      setting.setValue(path.join(homedir(), ".local", "bin", "scarb"));
+
+      await workbench.executeCommand("Cairo: Reload workspace");
```

Comment:
why are you reloading the workspace here? don't we have a clean state because we haven't opened any project yet?